src/departure-list/DeparturesList.tsx

```diff
@@ -31,7 +32,7 @@ export default function DeparturesList({
   isLoading = false,
   error,
   isInitialScreen,
-  currentLocation,
```

Comment:
Try to avoid `currentLocation` here, as the term is mainly used throughout the app for the current location of the app.

Reply:
Ah, think I got them all now 1a9825ef75144d93f94bad32fb243104c575d920.